src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -90,13 +89,14 @@ const getEmptyCoreStats = (): CompletionStats =>
 // This function is referenced in the recalculate-core-completion migration file.
 // Any code updates should validate that migration is unaffected.
 export const calcDonorCoreEntityStats = (
-  donor: Donor,
+  donor: Donor | DeepReadonly<Donor>,
```

Comment:
these are basically the same. this gets rid of type warnings. if we ignore typescript for moment and think about the function we likely either want to keep `donor` as read only or not as read only.